installer/windows/scripts/add_station_to_hosts.bat

```diff
@@ -0,0 +1,24 @@
+@ECHO OFF
+
+ECHO Executing add_station_to_hosts.bat
+
+SETLOCAL ENABLEDELAYEDEXPANSION
+
+@REM Adds station.massa redirection to Windows hosts file if not already present
+
+SET windows_hosts_file=%windir%\System32\drivers\etc\hosts
+
+@REM If station.massa is already in Windows hosts file, we can exit
+FINDSTR /c:"127.0.0.1 station.massa" "%windows_hosts_file%" >nul 2>&1
```

Comment:
Can you only check for "station.massa" to handle more complex nework scenarios ?

Reply:
Not fixed:

![image](https://github.com/massalabs/station/assets/16623956/d84fc32a-8c06-4436-8cdf-30f9572f6fb0)